spec.html

```diff
@@ -36847,7 +36879,7 @@ <h1>DetachArrayBuffer ( _arrayBuffer_ [ , _key_ ] )</h1>
           1. Return NormalCompletion(*null*).
         </emu-alg>
         <emu-note>
-          <p>Detaching an ArrayBuffer instance disassociates the Data Block used as its backing store from the instance and sets the byte length of the buffer to 0. No operations defined by this specification use the DetachArrayBuffer abstract operation. However, an ECMAScript implementation or host environment may define such operations.</p>
+          <p>Detaching an ArrayBuffer instance disassociates the Data Block used as its backing store from the instance and sets the byte length of the buffer to 0. No operations defined by this specification use the DetachArrayBuffer abstract operation. However, an ECMAScript host may define such operations.</p>
```

Comment:
But an implementation may not?

Reply:
Implementations certainly may. I can change this to "host or implementation". With an explicit sentence added to the host-defined definition that implementations are free to define any host-defined facility, is this still confusing?

Reply:
I would prefer "host or implementation" unless this sentence can somehow be reworded to use the term "host-defined" explicitly.

Reply:
This was reworded to "host or implementation"